src/7-to-8/summary.rst

```diff
@@ -413,110 +187,49 @@ managing disk space.
 
 These symlinks are created on a per install target basis, as configured in
 :cylc:conf:`global.cylc[install][symlink dirs]`. Install targets are managed on
-a site level, for more information see :ref:`Install Targets`.
+a site level, for more information see :ref:`Install Targets`
```

Comment:
The link when rendered ends in a `?`. Would you like me to re-write the title and have the sentence end in the `.`? Otherwise I think having both punctuation isn't right. 
![image](https://user-images.githubusercontent.com/37735232/155977737-a4d2a3fd-26b0-4e76-b74c-86fad881dbf1.png)